pom.xml

```diff
@@ -643,8 +643,6 @@
           <minimumJavaVersion>1.${java.level}</minimumJavaVersion>
           <systemProperties>
             <hudson.Main.development>${hudson.Main.development}</hudson.Main.development>
-            <!-- to avoid losing the focus on MacOS when a new forked jvm is created -->
-            <java.awt.headless>true</java.awt.headless>
```

Comment:
no reason to revert this is there?

Reply:
Probably not but without ability to actually test effects I prefer to cleanly revert at a PR level.